chart/templates/proxy_injector.yaml

```diff
@@ -168,10 +168,6 @@ data:
       value: tcp://0.0.0.0:{{.Values.InboundPort}}
     - name: LINKERD2_PROXY_DESTINATION_PROFILE_SUFFIXES
       value: {{.Values.ProfileSuffixes}}
-    - name: LINKERD2_PROXY_POD_NAMESPACE
-      valueFrom:
-        fieldRef:
-          fieldPath: metadata.namespace
```

Comment:
We'll stop using `proxy_injector.yaml` in #2334 but it appears you missed replacing this with `K8S_NS`?